### DISCUSSION TITLE
Help with Baseline Correction for SCR and Pupil Data in PsPM

### DISCUSSION THREAD

**Discussion by DaniyalBME** [bachlab/PsPM#781] [Q&A] [answered]
Hi PsPM Team

I hope you're all doing well. I'm reaching out for assistance with baseline correction for SCR and pupil data using PsPM. Could you please guide me on how to perform this correction within PsPM? If it's not possible within the software, any alternative suggestions would be greatly appreciated.

Thank you!

Best regards,
Daniyal

### Technical Info
* PsPM revision ID:7
* MATLAB version:2023b

**Comment by dominikbach** [comment, marked as answer]
Hi Daniyal.

all models implemented in PsPM consider an entire time series of data, rather than progressing trial-by-trial. Thus, no trial-by-trial baseline correction is required, and such functionality is also not implemented in PsPM. Session-wise baseline correction is modelled by an intercept parameter in the GLMs.

Dominik
